### ISSUE TITLE
Exporting edited character models not retaining changes

### ISSUE THREAD

**Issue by BoboLLC** [Grimrukh/soulstruct-blender#30]
**I apologize for my ignorance.**
**Thank You**

When I export my edited character model to the project folder, and reopen it using the import character button, all it brings up is the 
original character mesh. It's like my mesh isn't being saved for whatever reason.

  I thought it might've been an issue with the mesh not being rigged/parented with the armature, but that didn't seem to be it, as I didn't 
   add any new objects or meshes.
  I am unsure if this is an issue with my blender knowledge or if its an issue with the program itself.

My Process: 
 .Import Character model
 .Edit the mesh of the model
 .Save edited character model to project folder
 .Reopen the edited character model to see the mesh is back to being the original
 .Search for guides and find nothing except guides about maps
 .cry ;-;

I hope this is an issue of my ignorance of blender and that it can be solved easily.
pls help me i am willing to hop on a discord call or whatever else

**Comment by Grimrukh** [comment]
Hmm... have you ticked "Prefer Import from Project" in the Soulstruct general settings in Blender? Can you see the exported FLVER or CHRBND file in the project folder, and does it look different (e.g. different size) to the original game one?

**Comment by BoboLLC** [comment]
It appears that I did have the "Prefer Import from Project" box ticked, but the issue still occurred even when I made some new changes and attempted export. 
As for the file size, they were different.

Here is a screenshot of the file size difference : 
![image_2024-02-02_105246947](https://github.com/Grimrukh/soulstruct-blender/assets/108726830/1d471d27-c715-48a3-a894-17c8fd8b8131)
This was done with equipment instead and I had the same issue where changes to the mesh aren't being saved on export.

(With zero ill intent) I'm curious if you've used your tool to edit the chr/equipment meshes before, or is this new ground we're treading?
